.travis.yml

```diff
@@ -12,6 +12,7 @@ jobs:
       chmod +x minikube
       sudo mv minikube /bin/
       minikube config set vm-driver none
+      minikube config set kubernetes-version v1.17.3
```

Comment:
It would be cool to set this based on go.mod, so we don't need to remember to change this in the future. Maybe using go list?
```
$ go list -f '{{.Version}}' -m k8s.io/api
v0.17.3
```

Reply:
```suggestion
      minikube config set kubernetes-version $(go list -f '{{.Version}}' -m k8s.io/api | sed 's/^v0\./v1./')
```

Reply:
nice !!!